code/starfield/starfield.cpp

```diff
@@ -48,7 +47,7 @@ typedef struct {
 	vec3d pos;
 	int vclip;
 	float size;
-} motion_debris;
+} motion_debris_vclips;
```

Comment:
I would nominate this still be called `motion_debris` and the `SCP_vector<motion_debris>` below be called `motion_debris_info`, to vaguely match our naming distinction between static data and individual instance data in other places.

Reply:
I have a different struct called `motion_debris` and would prefer this one stay as `motion_debris_vclips` because it's more accurate. This particular struct is for each debris bitmap clip being rendered out of the max 300. `motion_debris` is the struct for what is now `Motion_debris_info`.

Reply:
I'm not going to push this too hard, but this struct isn't just the bitmap, it has everything specific an individual piece of... motion debris. It would be like calling `weapon` `weapon_bitmap` and `weapon_info` `weapon` or calling `ship` `ship_model` and `ship_info` `ship`. Not necessarily wrong per se, but there is a precedent for calling each individual thing which receives the graphics `thing`, and naming the more abstract class of them `thing_info`. Although I realize now each of these doesn't actually 'know' what type they are, but that's why I suggested the change.

Reply:
`Motion_debris` is now an array of `motion_debris_instance` struct objects. `Motion_debris_info` is now a vector of `motion_debris_type` struct objects.

Reply:
I'll take it.